service/src/test/java/tds/assessment/repositories/StrandQueryRepositoryImplIntegrationTests.java

```diff
@@ -0,0 +1,137 @@
+                        "   ('(SBAC_PT)SBAC-Mathematics-8-Spring-2013-2015', 'Strand3-key', 'Strand3', 0, 2, null),\n" +
+                        "   ('(SBAC_PT)SBAC-SEG1-MATH-8-Spring-2013-2015', 'Strand4-key', 'Strand4', 1, 3, null),\n" +
+                        "   ('(SBAC_PT)SBAC-SEG2-MATH-8-Spring-2013-2015', 'Strand5-key', 'Strand5', 0, 4, null)";
+
+        jdbcTemplate.update(tblSetOfAdminSubjectsInsertSQL1);
+        jdbcTemplate.update(tblSetOfAdminSubjectsInsertSQL2);
+        jdbcTemplate.update(tblSetOfAdminSubjectsInsertSQL2a);
+        jdbcTemplate.update(tblSetOfAdminSubjectsInsertSQL2b);
+        jdbcTemplate.update(insertStrandSQL);
+    }
+
+    @Test
+    public void shouldFindBothStrandsForNonSegmented() {
+        final String assessmentKey = "(SBAC_PT)IRP-Perf-ELA-11-Summer-2015-2016";
+        Set<Strand> strands = repository.findStrands(assessmentKey);
+        assertThat(strands).hasSize(2);
+
+        Strand strand1 = null;
+        Strand strand2 = null;
+
+        for (Strand strand : strands) {
+            if ("Strand1".equals(strand.getName())) {
+                strand1 = strand;
+            } else if ("Strand2".equals(strand.getName())) {
+                strand2 = strand;
+            }
+        }
+
+        assertThat(strand1.getKey()).isEqualTo("Strand1-key");
+        assertThat(strand1.getMinItems()).isEqualTo(2);
+        assertThat(strand1.getMaxItems()).isEqualTo(7);
+        assertThat(strand1.getSegmentKey()).isEqualTo(assessmentKey);
+        assertThat(strand1.getAdaptiveCut()).isEqualTo(-39.234F);
+        assertThat(strand2.getKey()).isEqualTo("Strand2-key");
+        assertThat(strand2.getMinItems()).isEqualTo(0);
+        assertThat(strand2.getMaxItems()).isEqualTo(4);
+        assertThat(strand2.getSegmentKey()).isEqualTo(assessmentKey);
+        assertThat(strand2.getAdaptiveCut()).isEqualTo(-37.432F);
+    }
+
+
+    @Test
+    public void shouldFindStrandsForAssessmentAndSegments() {
+        final String assessmentKey = "(SBAC_PT)SBAC-Mathematics-8-Spring-2013-2015";
+        Set<Strand> strands = repository.findStrands(assessmentKey);
+        assertThat(strands).hasSize(3);
+
+        Strand strand3 = null;
+        Strand strand4 = null;
+        Strand strand5 = null;
+
+        for (Strand strand : strands) {
+            if ("Strand3".equals(strand.getName())) {
+                strand3 = strand;
+            } else if ("Strand4".equals(strand.getName())) {
+                strand4 = strand;
+            } else if ("Strand5".equals(strand.getName())) {
+                strand5 = strand;
+            }
+        }
+
+        assertThat(strand3.getKey()).isEqualTo("Strand3-key");
+        assertThat(strand3.getMinItems()).isEqualTo(0);
+        assertThat(strand3.getMaxItems()).isEqualTo(2);
+        assertThat(strand3.getSegmentKey()).isEqualTo(assessmentKey);
+        assertThat(strand3.getAdaptiveCut()).isNull();
+        assertThat(strand4.getKey()).isEqualTo("Strand4-key");
```

Comment:
Same comment here and really throughout to have line breaks between assertions of different objects.